src/MetricsUploader/MetricsUploaderWindows.cpp

```diff
@@ -16,6 +16,7 @@ namespace orbit_metrics_uploader {
 constexpr const char* kSendLogEventFunctionName = "SendOrbitLogEvent";
 constexpr const char* kSetupConnectionFunctionName = "SetupConnection";
 constexpr const char* kShutdownConnectionFunctionName = "ShutdownConnection";
+constexpr const char* kClientLogFileSuffix = "Orbit";
```

Comment:
Can you add some more details to the commit message?
Where is that logfile stored? What is the actual filename? And where does the suffix come into play?

Reply:
Updated the commit message